### DISCUSSION TITLE
[storage] systemd.mount units not shown in Storage

### DISCUSSION THREAD

**Discussion by elisiariocouto** [cockpit-project/cockpit#20237] [General]
Hello everyone. Thank you for your effort on Cockpit, I've been using it for a few months and I always suggest friends to try it.

In my little debian box, I've setup a couple of CIFS mounts using [`systemd.mount`](https://www.freedesktop.org/software/systemd/man/latest/systemd.mount.html). I've chose this path since I needed to wait for a Wireguard tunnel on boot before mounting the shares and an easy way to solve this was to use systemd.mount with `After=mywireguard.service`.

Unfortunately, Cockpit is not showing these mounts on the Storage application. Are systemd.mount shares supported?

Thank you

**Comment by connelhooley** [comment]
I think this is covered in the following issues, hopefully it gets implemented one day:

- #15148
- #14260

Bootc makes using fstab much harder than mount units so hopefully that's a good usecase for the maintainers to consider.
